pyfiglet/__init__.py

```diff
@@ -81,6 +81,12 @@ class FontError(FigletError):
     """
 
 
+class InvalidColor(FigletError):
```

Comment:
I moved the exception here, and inherited from FigletError instead of Exception in order to match the other exception format